Department.php

```diff
@@ -0,0 +1,30 @@
+<?php
+    class Department
+    {
+        public function __construct(private string $name, private $Employees)
+        {
+        }
+        public function get_name(): string{
```

Comment:
Имена методов по PSR должны быть в camelCase. Также форматирование кода не по PSR
```suggestion
        public function getName(): string
        {
```